code/game/area/Space_Station_13_areas.dm

```diff
@@ -382,6 +397,14 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Central Primary Hallway"
 	icon_state = "hallC"
 
+/area/hallway/primary/central/ne
+
+/area/hallway/primary/central/nw
+
+/area/hallway/primary/central/se
+
+/area/hallway/primary/central/sw
+

```

Comment:
Is it worth giving these distinct names for when engineers have to go track down the relevant APCs?

Reply:
Not really something that /tg/ code does anywhere else. I'd like to see it happen, personally, but it's technically not a change this PR is about.

I'm fine with Purp adding it here or just making a **quick** PR after this one's merged to add the stuff to other maps.

Reply:
I like the unhelpfulness of "x has died in maintenance"

In the same vein of HELP MAINT...

Unuseful info